apps/examples/submitter.py

```diff
@@ -117,7 +123,7 @@ def main():
           update_config(tmp_config_path, f"  \"{name}\":", False if "collision" in sample else f"{apply},\n")
       
       update_config(tmp_files_config_path, "dataset = ", f"\"{dataset}\"\n")
-      update_config(tmp_files_config_path, "output_dir = ", f"\"{output_dir}\"\n")
+      update_config(tmp_files_config_path, f"{output_dir_name} = ", f"\"{output_dir}\"\n")
```

Comment:
I'm not sure I follow the logic here. Why would the config already have the `output_dir_name` value in it? And why would we need to replace it with `output_dir`?

Reply:
I did this so that I can later in the next step in the submit manager handle the case of output_hist_dir or output_trees_dir

Reply:
Still don't understand... ;) Let's discuss this one in person.